tests/radius/eap_test_server.c

```diff
@@ -0,0 +1,291 @@
+/*
+ * Example application showing how EAP server code from hostapd can be used as
+ * a library.
+ * Copyright (c) 2007, Jouni Malinen <j@w1.fi>
+ *
+ * This software may be distributed under the terms of the BSD license.
```

Comment:
Please add upstream source of this file (e.g. git commit and URL)

Reply:
ok